src/bundle.js

```diff
@@ -4,12 +4,6 @@
 // Use define from require.js not webpack's define
 var _define = window.define;
 
-// document-register-element
-var docRegister = require("document-register-element");
-_define("document-register-element", function() {
-    return docRegister;
-});
-
```

Comment:
why remove this and other piece?

Reply:
i don't know why this happening on firefox. removeing form bundle fix the problem.

Reply:
:confused: For me, only `webcomponents` caused an issue in Firefox.

Reply:
ahh i have found the way to fix